.github/pull_request_template.md

```diff
@@ -1,11 +1,33 @@
-## Thank you for your contribution! 
+<!-- This template is **just a guide**, delete any and all parts which you don't need! -->
 
-Please add one or two sentences _why_ this PR was made, and optionally what it adds/changes…
+## 📝 Docs
 
-## Checklist
+- [ ] Did you update the [CHANGELOG.md](https://github.com/criticalmaps/criticalmaps-ios/blob/master/CHANGELOG.md)?
 
-### Before merging the PR
+## 📲 What
 
-- [ ] If applicable, does the PR contain enough unit / UI tests?
-- [ ] If applicable, did you add a before / after screenshot of the feature?
-- [ ] Did you update the [CHANGELOG.md](https://github.com/criticalmaps/criticalmaps-ios/blob/master/CHANGELOG.md)?
+A description of the change.
+
+## 🤔 Why
+
+Some background context on why the change is needed.
+
+## 👀 See
+
+Screenshots, external resources?
```

Comment:
Do we want to add that (if possible) a Snapshot Test should be added and that is has to be recorded with an iPhone 8 (13.3)?

Reply:
Good idea. Got confused a couple of times with failed tests